index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio | Abenezer Fekadu</title>
+  <link rel="stylesheet" href="./style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    
+    
+      <nav class="navbar">
+
+        <header class="navbar-brand"><a href="home-page-link">Abenezer</a></header>
+    
+      
+        <ul class="navbar-list">
+          <li class="navbar-item">Portofolio</li>
+          <li class="navbar-item">About</li>
+          <li class="navbar-item">Contact</li>
+          <li class="navbar-item"><img src="./img/666162.png" alt="" class="nav-icon"></li>
+        </ul>
+      <div class="navbar-toggle">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+      </nav>
+      <section class="header">
+      <div class="header-text"><h1>Hey There. <span>I'm Abenezer</span> </h1>
+      </div>
+      <div class="body-text-header"><h2>I'm a Software Developer</h2>
+      </div>
+      <div class="body-text-main">
+        <p>I can help you build a product,feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
```

Comment:
Kindly stick to the Figma template and fix the font-family and size of the paragraph.

<img width="974" alt="Screenshot 2023-01-19 at 7 32 39 PM" src="https://user-images.githubusercontent.com/25887253/213502369-c7dce8b8-e15c-4710-9a61-a4856657e02f.png">
<img width="615" alt="Screenshot 2023-01-19 at 7 33 28 PM" src="https://user-images.githubusercontent.com/25887253/213502390-e2eec09b-c948-41f8-b1b6-1197c516087f.png">